packages/webapp/src/utils/api.tsx

```diff
@@ -128,29 +127,6 @@ export function useHostedSigninAPI() {
     };
 }
 
-export function useGetProvidersAPI(env: string) {
-    const signout = useSignout();
-
-    return async () => {
-        try {
-            const res = await apiFetch(`/api/v1/provider?env=${env}`);
-
-            if (res.status === 401) {
-                await signout();
-                return;
-            }
-
-            if (res.status !== 200) {
-                serverErrorToast();
-            }
-
-            return res;
-        } catch {
-            requestErrorToast();
-        }
-    };
-}
-
```

Comment:
don't you need to replace this endpoint in 2 steps? Otherwise if deploying server first the legacy endpoint won't be there anymore and if deploying the webapp first the new endpoint won't be available yet